Simplenote/Classes/SPTagsListViewController.m

```diff
@@ -371,6 +396,9 @@ - (BOOL)shouldSelectCellAtIndexPath:(NSIndexPath *)indexPath {
         case SPTagsListSectionTags: {
             return selectedTag == [self tagAtTableViewIndexPath:indexPath].name;
         }
+        case SPTagsListSectionBottom: {
+            return selectedTag == kSimplenoteUntaggedKey;
+        }
```

Comment:
I think you can avoid the `{ ... }` in the _SPTagsListSectionTags_ and _SPTagsListSectionBottom_, and return directly the value. Also in nested switch as well.

Reply:
Oh and the Hungarian Notation too 😊